tests/in/boids.wgsl

```diff
@@ -1,4 +1,5 @@
 const NUM_PARTICLES: u32 = 1500u;
+alias Vec2 = vec2<f32>;
```

Comment:
Again, much better to do this in `module-scope.wgsl` or `type-alias.wgsl`, which actually use the definition.